lib/IRGen/GenMeta.cpp

```diff
@@ -3858,11 +3868,14 @@ namespace {
     // Visitor methods.
 
     void addValueWitnessTable() {
+      if (IGM.getOptions().LazyInitializeClassMetadata)
+        return B.addNullPointer(IGM.WitnessTablePtrTy);
+
       // Without Objective-C interop, foreign classes must still use
       // Swift native reference counting.
       auto type = (IGM.ObjCInterop
-                   ? IGM.Context.TheUnknownObjectType
-                   : IGM.Context.TheNativeObjectType);
+                     ? IGM.Context.TheUnknownObjectType
```

Comment:
Try to avoid pointless whitespace changes; it makes the diff harder to review

Reply:
This diff is still present.

Reply:
Yeah, I will push the version that I have locally, which had the white space change undone.  I actually have been using the conversations to track the issues in the patches and mark them resolved when I get them taken care of.  I just have been trying to flush out some of the other patches so I haven't updated this just yet, but will try to do that tonight.